CONTRIBUTING.md

```diff
@@ -95,30 +98,91 @@ we would appreciate if your pull request applies to the following points:
 
   All committers, and all commits, are bound to
   the [Developer Certificate of Origin.](https://www.eclipse.org/legal/DCO.php)
-  As such, all parties involved in a contribution must have valid ECAs. Additionally commits can include
-  a ["Signed-off-by" entry](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
-* We prefer small changes and short living branches. Please do not combine independent things in one pull request.
+  As such, all parties involved in a contribution must have valid ECAs. Additionally, commits can 
+  include a ["Signed-off-by" entry](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
+* We prefer small changes and short living branches. Please do not combine independent things in one
+  pull request.
 
 
 * Excessive branching and merging can make git history confusing. With that in mind please
 
-    * [Squash your commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing)
+    * [squash your commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing)
       down to a few commits, or one commit, before submitting a pull request and
-    * [Rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) your pull request changes on top of the current
-      master. Pull requests shouldn't include merge commits.
+    * [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) your pull request changes on 
+      top of the current master. Pull requests shouldn't include merge commits.
 
 * Add meaningful tests to verify your submission acts as expected.
 
-* Where code is not self-explanatory add documentation providing extra clarification.
+* Where code is not self-explanatory, add documentation providing extra clarification.
 
-* Submit a draft pull request at early-stage and add people previously working on the same code as reviewer.
-  Make sure automatically checks pass before marking it as "ready for review":
+* Submit a draft pull request at early-stage and add people previously working on the same code as 
+  reviewer. Make sure automatically checks pass before marking it as "ready for review":
 
-    * _Intellectual Property Validation_ verifying the [Eclipse CLA](#eclipse-contributor-agreement) has been signed as
-      well as commits have been signed-off and
+    * _Intellectual Property Validation_ verifying the [Eclipse CLA](#eclipse-contributor-agreement) 
+      has been signed as well as commits have been signed-off and
     * _Continuous Integration_ performing various test conventions.
 
+## Project and Milestone Planning
+
+Milestones are planned for about 6 to 8 weeks. Future milestones describe an outlook on features to 
+be considered. The transparent process on GitHub enables participation. To decide what features and
+issues will be targeted, user requirements and sponsors are considered. Thus, everyone is able to
+influence the addressed developments and its priorities! 
```

Comment:
fits perfect to milestone section below, suggest to move it.

shortened version
```suggestion
We use milestones to set a common focus for a period of 6 to 8 weeks. 
The group of committers chooses issues based on customer needs and contributions we expect.
```